src/utils/Meta.js

```diff
@@ -15,7 +15,22 @@ const META = {
     element: 'element',
     view: 'view',
     module: 'module',
-  }
+  },
+
+  // library
+  isSemanticUI: ({_meta}) => _meta.library === META.library.semanticUI,
+  isStardust: ({_meta}) => _meta.library === META.library.stardust,
+
+  // type
+  isAddon: ({_meta}) => _meta.type === META.type.addon,
+  isGlobal: ({_meta}) => _meta.type === META.type.global,
+  isCollection: ({_meta}) => _meta.type === META.type.collection,
+  isElement: ({_meta}) => _meta.type === META.type.element,
+  isView: ({_meta}) => _meta.type === META.type.view,
+  isModule: ({_meta}) => _meta.type === META.type.module,
+
+  // parent
+  isChild: ({_meta}) => !!_meta.parent,
 };
 
 export default META;
```

Comment:
Defines meta data for components. Why library is it from, what type is it, does it have a parent?